utility/context.go

```diff
@@ -3,147 +3,124 @@ package utility
 import (
 	"encoding/hex"
 
-	"github.com/pokt-network/pocket/shared/codec"
 	"github.com/pokt-network/pocket/shared/modules"
 	typesUtil "github.com/pokt-network/pocket/utility/types"
 )
 
-// TODO: The implementation of `UtilityContext` should not be exposed.
-type UtilityContext struct {
-	bus     modules.Bus
-	Height  int64
-	Context *Context // IMPROVE: Rename to `persistenceContext` or `storeContext` or `reversibleContext`?
+type utilityContext struct {
+	bus    modules.Bus
+	height int64
 
-	// Data related to the Block being proposed
-	// TECHDEBT: When we consolidate everything to have a single `Block` object (a struct backed by a protobuf),
-	//           this can be simplified to just point to that object.
+	persistenceContext modules.PersistenceRWContext
+	// INVESTIGATE: Can we use the
```

Comment:
Can we use the.... 
![GeenoBttfDeloreanFluxcompensatorGIF](https://user-images.githubusercontent.com/29378614/218144796-962fe5ef-b937-4193-be91-1b45fbfc41f0.gif)

Please tell me that it's the flux capacitor, please please please!

Narrator: yes, he is old